engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java

```diff
@@ -1333,6 +1333,13 @@ public void validate() throws ConstraintViolationException
 
       OperatorMeta.PortMapping portMapping = n.getPortMapping();
 
+      // Check all attributes are serializable
+      for (Entry<Attribute<?>, Object> entry : n.getAttributes().entrySet()) {
```

Comment:
just as a practice - when doing validation - please report all the validation errors all at once wherever possible instead of one at a time.

Here: can you collect all the attribute serialization problems and report them at once?